src/main/java/com/theelixrlabs/healthcare/service/DoctorService.java

```diff
@@ -28,12 +31,12 @@ public DoctorService(DoctorRepository doctorRepository) {
     public DoctorDto saveDoctor(DoctorDto doctorDto) throws CustomException {
         validateDoctor(doctorDto);
         String aadhaarNumber = doctorDto.getAadhaarNumber();
+        if (doctorRepository.findByAadhaarNumber(aadhaarNumber).isPresent()) {
+            throw new CustomException(DoctorConstants.AADHAAR_ALREADY_PRESENT);
+        }
         String formattedAadhaarNumber = aadhaarNumber.substring(0, 4) + " " +
                 aadhaarNumber.substring(4, 8) + " " +
                 aadhaarNumber.substring(8, 12);
```

Comment:
Move the empty string into  constants and use it

Reply:
done ..